.circleci/ci/src/pipelines/tests/pipeline-pull-requests.spec.ts

```diff
@@ -0,0 +1,27 @@
+import * as fs from 'fs';
+import { generatePullRequestsConfig } from '../pipeline-pull-requests';
+
+describe('Pull requests workflow tests', () => {
+  it.each`
+    branchName                      | expectedFileName
+    ${'master'}                     | ${'pull-requests-master.yml'}
+    ${'4.1.x'}                      | ${'pull-requests-4-1-x.yml'}
+    ${'APIM-1234-my-custom-branch'} | ${'pull-requests-custom-branch.yml'}
+    ${'APIM-1234-run-e2e'}          | ${'pull-requests-run-e2e.yml'}
```

Comment:
Don't know if it's worth it... but I believe in the past there has been issues with the container creation if a branch is `some-prefix/apim-xx-branch`. At least with QA, they often use `test/apim-XX-branch`.  Do you think we should add a test?

Reply:
good point. Mergify uses also this syntax I think. I will add a test.